brayns/Brayns.cpp

```diff
@@ -1076,9 +1086,7 @@ Brayns::Brayns(int argc, const char** argv)
 {
 }
 
-Brayns::~Brayns()
-{
-}
+Brayns::~Brayns() {}
```

Comment:
= default in the header

Reply:
This needs to be in the cpp file due to the pimpl pattern.